xrootd.sh

```diff
@@ -41,8 +42,10 @@ esac
 
 rsync -a --delete $SOURCEDIR/ $BUILDDIR
 
-[ x"$XROOTD_V4" = x"True" ] && sed -i.bak 's/"uuid.h"/"uuid\/uuid.h"/' $(find . -name "*Macaroon*Handler*.cc")
-
+if [ x"$XROOTD_V4" = x"True" ]; then
+    sed -i.bak 's/"uuid.h"/"uuid\/uuid.h"/' $(find . -name "*Macaroon*Handler*.cc")
+    sed -i.bak '/^\s*--force-reinstall \\/s/--force-reinstall/--force-reinstall --ignore-installed/' "${BUILDDIR}/bindings/python/CMakeLists.txt"
```

Comment:
the intent is to not see/used already installed packages, as it can happen to have system-side installed packages that one might not want or cannot remove. 
So, with the context that there can be system installed xrootd and/or `--user` install
which must not be seen by the alibuild/alienv environment, what other solution do you see?

Reply:
> So, with the context that there can be system installed xrootd and/or --user install
which must not be seen by the alibuild/alienv environment, what other solution do you see?

Apologies, as I guess I'm confused. I thought you were trying to _replace the system installed_ XRootD with this. If you're not, then I don't understand what `--force-reinstall --ignore-installed` is trying to do.

Though you don't need to explain your system to me, so I would recommend ignoring any comments from me if things are working to an extent that you're happy with here. :+1: